src/api.js

```diff
@@ -9,6 +9,11 @@ export const MOCK_MEMBER = {
 export const MOCK_SELF = { ...MOCK_MEMBER, ...({ lang: 'en_US' }) };
 export const MOCK_SELF_FR = { ...MOCK_MEMBER, ...({ lang: 'fr_FR' }) };
 
+export const MOCK_PAYLOAD = {
```

Comment:
sorry to be nitpicky about this, but "payload" is a overloaded term in Node-land (`request.payload`, `response.payload`), so can this be `MOCK_FETCH_API_RESPONSE`?

Also, this mock is specific to some data manipulation inside the application (`fetchUtils` combines the `/api` endpoint "responses" with the original queries), not from the REST API, which is what other mocks in this module correspond to, so it should move over to `src/app.js`.